src/Appwrite/Platform/Workers/Usage.php

```diff
@@ -4,22 +4,22 @@
 
 use Exception;
 use Utopia\CLI\Console;
-use Utopia\Database\Database;
-use Utopia\Database\DateTime;
 use Utopia\Database\Document;
 use Utopia\Platform\Action;
+use Appwrite\Event\UsageDump;
 use Utopia\Queue\Message;
 
 class Usage extends Action
 {
-    protected static array $stats = [];
-    protected array $periods = [
-        '1h' => 'Y-m-d H:00',
-        '1d' => 'Y-m-d 00:00',
-        'inf' => '0000-00-00 00:00'
-    ];
-
-    protected const INFINITY_PERIOD = '_inf_';
+    private array $stats = [];
+    private int $lastTriggeredTime = 0;
+    private int $keys = 0;
+    private const INFINITY_PERIOD = '_inf_';
+    private const KEYS_THRESHOLD = 5;
+    private const KEYS_SENT_THRESHOLD = 60;
```

Comment:
Lets use this  variable we already have
 $interval = (int) App::getEnv('_APP_USAGE_AGGREGATION_INTERVAL', '60000');